src/main/resources/com/google/api/codegen/csharp/gapic_csproj.snip

```diff
@@ -0,0 +1,62 @@
+# packageInfo: PackageInfoView
+@snippet generate(packageInfo)
+  <?xml version="1.0" encoding="utf-8"?>
+  <Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+
+      <!-- TODO: Version defaults to 1.0.0, edit as required -->
+      <Version>{@packageInfo.version}</Version>
+      
+      <!-- TODO: Nuget packaging option -->
```

Comment:
Nit: NuGet :)

And "options" instead of "option"?

Reply:
Done